app/views/layouts/application.html.haml

```diff
@@ -14,7 +14,7 @@
     %meta{:name => "author", :content => "Diaspora, Inc."}/
 
     %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
-    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}
+    %link{:rel => 'apple-touch-icon', :href => "#{image_path('favicon.png')}"}
```

Comment:
You think the favicon is better used here? Note that it's used by many mobiles for example if you create a shortcut on your drawer.